networking/src/Ntp/Client.hs

```diff
@@ -298,6 +298,28 @@ spawnNtpClient settings ncStatus = do
     fn :: Last (WithAddrFamily t Socket.Socket) -> IO ()
     fn (Last sock) = Socket.close $ runWithAddrFamily sock
 
+    -- Try to resolve addresses, on failure wait 30s and start again.
+    resolve = do
+        logInfo "resolve DNS"
```

Comment:
This will get an `ntp` prefix from the logger name yes?

Reply:
Yes, seen as `NtpClient` in the logs.